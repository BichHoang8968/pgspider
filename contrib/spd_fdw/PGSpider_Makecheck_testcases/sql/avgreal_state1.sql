SELECT AVG(i) FROM avgreal; 'avgreal'); 
