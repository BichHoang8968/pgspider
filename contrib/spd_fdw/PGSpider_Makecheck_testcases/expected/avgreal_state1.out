4.34751470926014e+24